src/enrichment_facility_tests.cc

```diff
@@ -259,175 +515,10 @@ TEST_F(EnrichmentFacilityTest, ValidReq) {
-  using cyclus::Converter;
-  using cyclus::Material;
-
-  // a request is made for the current available inventory amount
-
-  std::set<RequestPortfolio<Material>::Ptr> ports =
-      src_facility->GetMatlRequests();
-
-  ASSERT_EQ(ports.size(), 1);
-  ASSERT_EQ(ports.begin()->get()->qty(), inv_size);
-
-  const std::vector<Request<Material>*>& requests =
-      ports.begin()->get()->requests();
-  ASSERT_EQ(requests.size(), 1);
-
-  Request<Material>* req = *requests.begin();
-  EXPECT_EQ(req->requester(), src_facility);
-  EXPECT_EQ(req->commodity(), in_commod);
-
-  const std::set< CapacityConstraint<Material> >& constraints =
-      ports.begin()->get()->constraints();
-  EXPECT_EQ(constraints.size(), 0);
-}
-
-// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(EnrichmentFacilityTest, Extract) {
-  using cyclus::Material;
-  cyclus::Env::SetNucDataPath();
-  double qty = 1000;  // 5 kg
-  Material::Ptr base = GetMat(qty);
-  double time = tc_.get()->time();
-      // cyclus::Material::Create(src_facility, qty,
-      //                                   tc_.get()->GetRecipe(in_recipe));
-  Material::Ptr base2 = GetMat(qty);
-  base->Absorb(base2);
-  double product_assay = 0.05;  // of 5 w/o enriched U
-  cyclus::CompMap v;
-  v[922350000] = product_assay;
-  v[922380000] = 1 - product_assay;
-  // target qty need not be = to request qty
-  Material::Ptr target = cyclus::Material::CreateUntracked(
-      5, cyclus::Composition::CreateFromMass(v));
-  Material::Ptr response = base->ExtractComp(6, target->comp());
-}
-
-// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(EnrichmentFacilityTest, Accept) {
-  using cyclus::Bid;
-  using cyclus::Material;
-  using cyclus::Request;
-  using cyclus::Trade;
-
-  // an enrichment facility gets two trades, each for 1/3 of its inv size
-  // note that comp != recipe is covered by AddMat tests
-  // note that qty >= inv capacity is covered by toolkit::ResourceBuff tests
-
-  double qty = inv_size / 3;
-  std::vector< std::pair<cyclus::Trade<cyclus::Material>,
-                         cyclus::Material::Ptr> > responses;
-
-  Request<Material>* req1 =
-      Request<Material>::Create(DoRequest(), src_facility, in_commod);
-  Bid<Material>* bid1 =
-      Bid<Material>::Create(req1, GetMat(qty), trader);
-
-  Request<Material>* req2 =
-      Request<Material>::Create(DoRequest(), src_facility, in_commod);
-  Bid<Material>* bid2 = Bid<Material>::Create(req2, GetMat(qty), trader);
-
-  Trade<Material> trade1(req1, bid1, qty);
-  responses.push_back(std::make_pair(trade1, GetMat(qty)));
-  Trade<Material> trade2(req2, bid2, qty);
-  responses.push_back(std::make_pair(trade2, GetMat(qty)));
-
-  EXPECT_DOUBLE_EQ(0.0, src_facility->InventorySize());
-  src_facility->AcceptMatlTrades(responses);
-  EXPECT_DOUBLE_EQ(qty * 2, src_facility->InventorySize());
-
-  delete bid2;
-  delete bid1;
-  delete req2;
-  delete req1;
-}
```

Comment:
Tested that incoming material was accurately added to inventory - replaced with RequestQty